internal/cli/configuration/build_cache.go

```diff
@@ -32,3 +36,22 @@ func (s *Settings) GetBuildCacheTTL() time.Duration {
 	}
 	return s.Defaults.GetDuration("build_cache.ttl")
 }
+
+// GetBuildCachePath returns the path to the build cache.
+func (s *Settings) GetBuildCachePath() (*paths.Path, bool) {
+	p, ok, _ := s.GetStringOk("build_cache.path")
+	if !ok {
+		return nil, false
+	}
+	return paths.New(p), true
+}
+
+// GetBuildCacheExtraPaths returns the extra paths to the build cache.
+// Those paths are visited to look for precompiled items if not found elsewhere.
+func (s *Settings) GetBuildCacheExtraPaths() paths.PathList {
+	var res paths.PathList
+	if p, ok, _ := s.GetStringSliceOk("build_cache.extra_paths"); ok {
+		return paths.NewPathList(p...)
+	}
+	return res
```

Comment:
```suggestion
	if p, ok, _ := s.GetStringSliceOk("build_cache.extra_paths"); ok {
		return paths.NewPathList(p...)
	}
	return nil
```
I think we can remove the `res` variable since it's unused and only returned.